temporian/implementation/numpy_cc/operators/tick_calendar_utils_test.cc

```diff
@@ -0,0 +1,19 @@
+
+#include "temporian/implementation/numpy_cc/operators/tick_calendar_utils.h"
+
+#include "gtest/gtest.h"
+
+TEST(IsLeapYear, Base) {
+  EXPECT_TRUE(IsLeapYear(1972));
+  EXPECT_FALSE(IsLeapYear(1970));
+}
+
+TEST(UTCMkTime, Base) {
+  // Tested with www.epochconverter.com
+  EXPECT_EQ(UTCMkTime(1970, 1, 1, 0, 0, 0).value().seconds_since_epoch, 0);
```

Comment:
Can you add a few more tests, e.g. a test before 1970 and a test after the year 3000? And one in Summer time and one in winter time

Reply:
And one that's on February 29

Reply:
*Added all requested test and a few others. Added test for invalid dates.